.yamllint

```diff
@@ -42,7 +42,7 @@ rules:
   key-duplicates: enable
   key-ordering: disable
   line-length:
-    max: 100
+    max: 120
```

Comment:
According to our ADR ([here](https://www.notion.so/cardinalcryptography/Workflow-and-GHA-naming-conventions-34f9d23c8136457184fb77fb169ba573)) - "The column limit is 100 characters, as per our current rustmt settings".

Reply:
jeez, are you people working on calculators?

Reply:
If you really cannot split some lines (i.e. link to the some website) you can use this method to temporarily bypass this rule like here :
```        # yamllint disable rule:line-length
        run: |
          wget https://github.com/WebAssembly/binaryen/releases/download/version_110/binaryen-version_110-x86_64-linux.tar.gz \
            -O /tmp/binaryen-version_110-x86_64-linux.tar.gz
          tar -zxvf /tmp/binaryen-version_110-x86_64-linux.tar.gz -C /tmp
          sudo chmod +x /tmp/binaryen-version_110/bin/*
          sudo cp -rf /tmp/binaryen-version_110/* /usr/
        # yamllint enable rule:line-length```